From c25ca2d0487249a8b64dfdcda4e7d88179dea75a Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Thu, 3 Sep 2020 10:13:05 +0100
Subject: [PATCH] Export vte_pty_spawn_finish again

This was accidentally unexported in af649664, breaking the ABI.

Signed-off-by: Simon McVittie <smcv@debian.org>
Resolves: https://gitlab.gnome.org/GNOME/vte/-/issues/278
---
 src/vte/vtepty.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/vte/vtepty.h b/src/vte/vtepty.h
index 875119b7..5d1fb38f 100644
--- a/src/vte/vtepty.h
+++ b/src/vte/vtepty.h
@@ -127,6 +127,7 @@ void vte_pty_spawn_with_fds_async(VtePty *pty,
                                   GAsyncReadyCallback callback,
                                   gpointer user_data) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1) _VTE_GNUC_NONNULL(3);
 
+_VTE_PUBLIC
 gboolean vte_pty_spawn_finish(VtePty *pty,
                               GAsyncResult *result,
                               GPid *child_pid /* out */,
-- 
GitLab

